src/Controls/src/Core/Page/Page.cs

```diff
@@ -684,8 +688,10 @@ public void SendDisappearing()
 
 			_hasAppeared = false;
 
+#pragma warning disable CS0618 // TODO: Remove this API in .NET 11.
```

Comment:
Link to issue for removal.

Reply:
@mattleibow / @jsuarezruiz, I have created a new issue report https://github.com/dotnet/maui/issues/30155 for this and linked it.